packages/web-components/src/components/notice-choice/notice-choice.ts

```diff
@@ -0,0 +1,595 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { checkPreferencesv3, loadContent } from './services';
+import { customElement, html, LitElement, property } from 'lit-element';
+import {
+  emailRegExp,
+  getMappedValue,
+  getNcContentFromWindow,
+  resetToWorldWideContent,
+} from './utils';
+
+import countrySettings from './country-settings';
+import ddsSettings from '../../internal/vendor/@carbon/ibmdotcom-utilities/utilities/settings/settings';
+import settings from 'carbon-components/es/globals/js/settings';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+import styles from './notice-choice.scss';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import { worldWideContent } from './world-wide-content';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Notice Choice
+ *
+ * @element dds-notice-choice
+ */
+@customElement(`dds-notice-choice`)
+class NoticeChoice extends StableSelectorMixin(LitElement) {
+  /**
+   * properties for passed attributes.
+   */
+  @property({ type: String, reflect: true, attribute: true })
+  questionchoices = '1';
+
+  @property({ type: String, attribute: true })
+  country = 'US';
+
+  @property({ type: String, attribute: true })
+  state = '';
+
+  @property({ type: String, attribute: true })
+  locale = 'us-en';
+
+  @property({ type: String, attribute: true })
+  termsConditionLink = html``;
+
+  @property({ type: String, attribute: true })
+  bpidLegalText = html``;
+
+  @property({ type: Boolean, attribute: true })
```

Comment:
Each of these properties should be used as attribute values, and hyphenated instead of camel-casing where needed, i.e. `attribute: 'question-choices'`